modules/library/coverage/src/main/java/org/geotools/coverage/grid/io/imageio/geotiff/GeoTiffConstants.java

```diff
@@ -147,7 +147,7 @@
 
 	public static final int UNDEFINED = 0;
 	
-	public static TIFFTag NODATA_TAG = new TIFFTag("GDAL_NODATA", GeoTiffConstants.TIFFTAG_NODATA, 
+	public static TIFFTag nodataTag = new TIFFTag("GDAL_NODATA", GeoTiffConstants.TIFFTAG_NODATA, 
```

Comment:
Not sure what is happening to this one, we are not in position to change a public API (many of your other examples are changing private static fields).

Consider checking if this field could be marked final instead of breaking API.